### DISCUSSION TITLE
Is export of projects from Android just broken/incomplete

### DISCUSSION THREAD

**Discussion by AveryAndrews** [opengisch/QField#5541] [Q&A]
Attempting to export a project to outside of the Android protected area, I usually (not always) get an incomplete result, missing the .qgs file among (very many) other things. Prior to the Android update that made it impossible for many apps to access the Android area,  I rarely did this, but don't recall any such problem.  The doc at https://docs.qfield.org/get-started/tutorials/get-started-qfs/ describes export with iOS but not Android.

**Comment by SeqLaz** [comment]
Hi @AveryAndrews, indeed, you can access and extract information from the Android device by connecting it to your computer via USB. The path is `<drive>:/Android/data/ch.opengis.qfield/Imported Projects/{project_name}`.

**Reply by AveryAndrews** [reply]
Thanks; I know how to do this, but it might be better not to have to rely on USB, and, meanwhile, the export project option exists in the interface but appears to only partially functional (unless there's a fine point I'm missing, which is entirely possible).

**Reply by nirvn** [reply]
@AveryAndrews , can you provide a video recording of the steps you are taking to export the project folder? Thanks

**Reply by AveryAndrews** [reply]
I will try to figure out a way to do that (getting usable video of me using the phone with QField on it might be tricky), but I could give series of screenshots with steps plus the final result.

**Reply by BTreeTech** [reply]
On Android I have tested the below and it seems to work. 
COG icon>Home>Open Local File>Imported Projects> 3 dots> send compressed folder to...
![image](https://github.com/user-attachments/assets/d9e7292c-714d-40db-9f5d-d173504804dc)

**Comment by AveryAndrews** [comment]
Sending compressed folders seemed to work when I tried it, but what I'm complaining about does not always happen.  So if you take BtreeTech's sequence, but choosing 'export to folder ...', then I sometimes, but not always (but always, recently, for this particular project) get the severely reduced result:

![Screenshot_20240812-185208](https://github.com/user-attachments/assets/797a4f96-be3a-4559-82dd-828b1d763d0f)

which lacks the .qgs file and much else.

Here is a screenshot of the files in the folder as seen via USB:

![path_screenie](https://github.com/user-attachments/assets/c3f4d41a-b3a6-4590-83bb-998442e908c8)

The phone is a Google Pixel 8 (not Pro)

**Comment by AveryAndrews** [comment]
What seems to be going on is that the QField seems to take a rather long time to export a project, somewhere between 2 and 5 minutes for a reliable result for my 1.4G project (many photos), and looking at the result too soon with something such as File Manager + seems to interrupt the process, so that it doesn't restart on its own.  But FM+ can copy the full project to somewhere else in about 15 secs.  Re-doing the export to the same folder and leaving the app open and doing nothing with the phone for a few minutes seems to be able to complete the export.

Which seems to me to be an 'issue', if not a full-fledged bug.
